manifests/plugin/foreman_gutterball.pp

```diff
@@ -1,12 +1,18 @@
 # foreman_gutterball plugin
 class katello_devel::plugin::foreman_gutterball {
   $path = "${katello_devel::deployment_dir}/foreman-gutterball"
+  $foreman_settings_path = "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d"
 
   vcsrepo { $path:
     ensure   => present,
     provider => git,
     source   => 'https://github.com/Katello/foreman-gutterball.git',
     user     => $katello_devel::user
+  } ~>
+  class { 'katello::plugin::foreman_gutterball_config':
+    foreman_plugins_dir => "${foreman_settings_path}/gutterball.yml",
```

Comment:
the filename should be (afaik) the same as the name of the plugin, so we'll want `foreman_gutterball.yaml` here.